package.json

```diff
@@ -31,5 +31,8 @@
     "eslint": "^3.2.2",
     "expect": "^1.20.2",
     "mocha": "^3.0.2"
+  },
+  "dependencies": {
+    "saul": "0.0.17"
```

Comment:
`saul` should be a `devDependency` here.

Otherwise it will get installed into `node_modules` on a consuming project when the consuming project does a `yarn install` or `npm i`